### DISCUSSION TITLE
getMILPParams is not specified

### DISCUSSION THREAD

**Discussion by dimwas** [SysBioChalmers/RAVEN#574] [Help me!] [answered]
I am currently trying to use the `fillGaps` function. In the function's description, there is a following line:

`%   params              parameter structure as used by getMILPParams (opt)`

I am not sure what was meant here. I did not find the `getMILPParams` function in RAVEN. I suppose this params structure can allow me, for example, to increase a time threshold for gapfilling, so I would very much like to know how to construct it.

**Comment by edkerk** [comment, marked as answer]
`getMILPParams` was removed in release 2.7.0, instead the parameters are directly coded in `optimizeProb`:

https://github.com/SysBioChalmers/RAVEN/blob/cd82b5c6ba26a88bed2a9ca179c682a6434fd541/solver/optimizeProb.m#L46-L57

Perhaps not ideal, but e.g. time thresholds can still be adjusted there.
